.travis.yml

```diff
@@ -29,12 +29,15 @@ install:
   - conda info -a
 
   # install dependencies
-  - conda install numpy scipy astropy six pip pyyaml line_profiler pycodestyle coveralls h5py pytest-cov psutil
-  - conda install -c conda-forge mpi4py
-  - pip install git+https://github.com/RadioAstronomySoftwareGroup/pyuvdata.git
-  - python setup.py install
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION coveralls
+  - source activate test-environment
+  - conda env update -n test-environment -f environment.yml
+  - pip install .
   - python -c "from pyuvdata import version; version.main()"
   - conda list
+  # removed this, as it's installed via conda. However, it may be that we *want* it to
+  # be from github...
+  #- pip install git+https://github.com/RadioAstronomySoftwareGroup/pyuvdata.git
```

Comment:
Let's remove this commented out section. There's always the history, we can get back to it if needed.

Reply:
Done in ec6d28e450cd4c960a4b71072ca4edf36ab09f28